src/store.tsx

```diff
@@ -0,0 +1,117 @@
+import { useReducer, useEffect, createContext, useContext, useCallback, useMemo, useState } from "react";
+
+export interface Movie {
+  id: string;
+  name: string;
+  genres: string[];
+  year: number;
+  poster: string;
+}
+
+function useMoviesSource(): {
+  movies: Movie[];
+  filterBy: string;
+  setFilterBy: (filterBy: string) => void;
+  sortBy: string;
```

Comment:
Let's give `sortBy` a type

Reply:
Continued in [the next comment](https://github.com/fedaipaca/movies-client-reactjs/pull/4#discussion_r1038993533). This thread is resolved